dipy/reconst/rumba.py

```diff
@@ -718,10 +820,14 @@ def rumba_deconv_global(data, kernel, mask, n_iter=600, recon_type='smf',
     -------
     fodf_wm : 4d ndarray (x, y, z, M-1)
         fODF computed for each voxel.
-    f_iso : 3d ndarray (x, y, z)
-        Isotropic volume fraction at each voxel.
+    f_gm : 3d ndarray (x, y, z)
+        Grey matter volume fraction at each voxel.
```

Comment:
super minor: for consistency, can you replace grey matter with GM everywhere in the file